### ISSUE TITLE
Bus stop with shelter

### ISSUE THREAD

**Issue by spindr** [gravitystorm/openstreetmap-carto#3124]
Bus stop has an icon, shelter has an icon -- is it possible to combine them to depict a bus stop with a shelter. Obviously not a high priority -- but might be good to see them differentiated -- especially as it's one of the StreetComplete challenges.

**Comment by james2432** [comment]
![path3716](https://user-images.githubusercontent.com/498547/37562171-ae690c54-2a37-11e8-96dd-eb979173d362.png)
Like this?

**Comment by HolgerJeromin** [comment]
This could be an shelter_type=public transport icon
But not the bus_stop icon

**Comment by james2432** [comment]
What tag are we trying to add?
https://wiki.openstreetmap.org/wiki/Tag:highway=bus_stop ? Seems like public transport to me
Unless you mean this, but with a shelter?
https://wiki.openstreetmap.org/wiki/Tag:amenity=bus_station

**Comment by spindr** [comment]
I was thinking something like the attached.

Cheers,

Neil

**Comment by james2432** [comment]
Missing the attachment

**Comment by spindr** [comment]
Apologies - trying to include it directly in the email

Neil

bus stop with shelter png

**Comment by kocio-pl** [comment]
GitHub probably removes them in the comments. You should try with gist: 

https://help.github.com/articles/creating-gists/

**Comment by spindr** [comment]
Hope this works:
https://gist.github.com/spindr/38b2fc8f9ee5eef389fd76d3ff3a9791/e37143278246e02b8dfcf9272c557b8483ba9625
-- I had to click [View] next to the PNG file.

Cheers,
Neil

**Comment by kocio-pl** [comment]
Yes, it is visible now, but it should be recgnizable at 14 px - could you try with his resolution? Most good icon designs fails this check.

**Comment by james2432** [comment]
i  tried combining them before..problem is at 14x14 it looks like garbage

**Comment by matthijsmelissen** [comment]
I think this is too specific for a general purpose map.

**Comment by kocio-pl** [comment]
Related to #2431.

**Comment by geesona** [comment]
Here is a proposed icon for a bus stop with a shelter.
<script src="https://gist.github.com/geesona/a7723f34fc41145d38f8ff37e69ffa99.js"></script>
It will need tidy up.

**Comment by geesona** [comment]
This is the same as the previous icon but saved as a simple svg.
https://gist.github.com/geesona/1e04cad1f943a3fb54ab04761676f2cd

**Comment by geesona** [comment]
Hi - Is anybody there ?

**Comment by kocio-pl** [comment]
Yes. Unfortunately this image is not clear for me.

**Comment by geesona** [comment]
As this icon is both a bus shelter and a bus stop could it be slightly larger than the 14 x14? My previous attempt is a slightly squashed existing bus stop icon with a cover over it.  It may be easier to see if it was the same size bus stop as the existing bus stop with a cover over it?
Or how about a man with his arm out under a shelter but then it all gets a bit confusing!

https://gist.github.com/geesona/1fc0b55ea9be965fef6df78a84c071db

**Comment by jeisenbe** [comment]
Related to #2431
